src/main/java/uk/gov/ons/ctp/response/action/service/impl/decorator/ActionRequestDecoratorContextFactory.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.ons.ctp.response.action.service.impl.decorator;
+
+import uk.gov.ons.ctp.response.action.domain.model.Action;
+
+public interface ActionRequestDecoratorContextFactory {
```

Comment:
Feels like the Decorator can be dropped from the class name

Reply:
Done